### ISSUE TITLE
[Docs]: Fallback search

### ISSUE THREAD

**Issue by dbowling** [chrisgrieser/shimmering-obsidian#57]
### Change Requested

I just set up shimmering-obsidian as one of my Alfred fallback searches and I didn't see this documented. The process was pretty simple and just required a quick modification to the 'o' quick search by adding a trigger. 

Is this something you'd be open to a pull request for? I would like to update the docs and add the trigger to the workflow. I don't know that this setup can (or should) be automated, but if that is desired I can do some research and include that too.

Here's what the changes look like:

<img width="599" alt="02-11-2022  - Friday -  Alfred Preferences Alfred Preferences - LXMzJeSH" src="https://user-images.githubusercontent.com/46435/153664927-afe4499a-99e0-4e31-9b1d-1ebcbe4924bb.png">

<img width="905" alt="02-11-2022  - Friday -  Alfred Preferences Alfred Preferences - CQONyzfw" src="https://user-images.githubusercontent.com/46435/153665052-97c221ea-50c0-444d-bafe-0bbef3cb09ca.png">
 

### Link to the relevant section of the documentation

https://github.com/chrisgrieser/shimmering-obsidian/blob/main/docs/Workflow%20Configuration.md

**Comment by chrisgrieser** [comment]
Good idea, I added the fallback search and added to the documentation. Thanks!
https://github.com/chrisgrieser/shimmering-obsidian/blob/main/docs/Alfred-based%20Quick%20Switcher.md#vault-search-as-alfred-fallback
